connector/signaltometricsconnector/metadata.yaml

```diff
@@ -0,0 +1,30 @@
+type: signaltometrics
+scope_name: github.com/elastic/opentelemetry-collector-components/connector/signaltometricsprocessor
+
+status:
+  class: connector
+  stability:
+    alpha: [traces_to_metrics, logs_to_metrics, metrics_to_metrics]
+  distributions: []
+
+tests:
+  config:
+    spans:
+      - name: span.duration.histogram
+        description: "Span duration as histogram"
+        histogram:
+          value: Microseconds(end_time - start_time)
+      - name: span.duration.exponential_histogram
+        description: "Span duration as exponential histogram"
+        exponential_histogram:
+          value: Microseconds(end_time - start_time)
+    logs:
+      - name: logrecord.count
+        description: "Count of log record"
+        sum:
+          value: Int("1") # Count each log record as 1
```

Comment:
Is this equivalent to the default? I.e. can you just do `sum: {}`? Also, can you instead do `value: 1`, or does it need to Int converter?

Reply:
I didn't add the default behavior for `sum` metric so far. With histograms, it is implicit what count would be but for sum metric, it didn't feel right to assume the behavior. No strong objection to having the defaults though.

> Also, can you instead do value: 1, or does it need to Int converter?

We can do `value: "1"` as it has to be a string but we don't need the int converter - that is remnant from my debugging something. I will simplify these.